### ISSUE TITLE
Automated summaries + releases via GitHub Actions?

### ISSUE THREAD

**Issue by niemasd** [globaldothealth/ebola#13]
Given the real-time urgency of this, it might be good to have automated summaries + releases via GitHub Actions. I'm not sure what summaries would be most useful for folks using the data, but please feel free to ping me if you want any help with any specific ideas that may come up.

Regarding automated releases, the following GitHub Actions from one of my repos might be a helpful template:

https://github.com/niemasd/curriculum-vitae/blob/master/.github/workflows/compile.yml

**Comment by jim-sheldon** [comment]
Hello,

Thanks for your input! Hearing from users like yourself helps us decide on features and prioritization.

I would like to proceed by separating the feature you would like from the way we provide it.

The what:
Any data sets you would find useful, ask for, and I will raise it with the team.
For our latest data set, and historical data sets, we [link to our S3 bucket](https://3mmuwilir3.execute-api.eu-central-1.amazonaws.com/web) from our README. We can add more data sets to that bucket and provide open access.
Eventbridge drives a Batch job that updates our database and bucket. Right now it runs every four hours M-F, so it will pick up changes often.

If you are asking us to generate LaTex files specifically, I am not as sure. I would need to talk with my team.

The how:
Whether it's automating data ingestion or making LaTex files, Github is the wrong tool for the job. As an engineer I need to consider many factors (e.g. collecting metrics for Ops). I tried Github Actions, and they work fine for some work, but not this (see [this issue](https://github.com/globaldothealth/monkeypox/issues/213)). The Eventbridge-Batch pattern helps our team satisfy more requirements and is consistent with software industry standards and practices.

In short, I am not an epidemiologist or scientist, I only support those communities, and I am responsible and accountable for maintaining and updating our systems. You tell me what you need, I will find how to do it.

Let me know what you think, and thanks again!

**Comment by jim-sheldon** [comment]
Great connecting with you over email.
Going to close this based on our conversation, but feel free to reopen this or open new issues whenever you like!
